command/delegate/delegate.go

```diff
@@ -123,7 +123,14 @@ func (c *delegateCommand) run(*kingpin.ParseContext) error {
 		KeyFile:             config.DefaultPoolSettings.KeyFile,
 	}
 	// process the pool file
-	pools, poolFileErr := cloudaws.ProcessPoolFile(c.awsPoolfile, &c.defaultPoolSettings)
+	//pools, poolFileErr := cloudaws.ProcessPoolFile(c.awsPoolfile, &c.defaultPoolSettings)
+	//if poolFileErr != nil {
```

Comment:
ignore for now, will be fixed when we update to support one pool file

Reply:
yes, this is the correct implementation for now.. see my comment in the daemon.go

Reply:
I've uncommented this for now - will pick up work after merge